### ISSUE TITLE
Adding a new graphql api using Visual Studio Code fails

### ISSUE THREAD

**Issue by gsans** [aws-amplify/amplify-cli#2181]
**Describe the bug**
During a workshop few attendees experienced this issue which seems to be linked with guided schema creation.

**To Reproduce**
1. amplify add api
2. ? Do you want a guided schema creation? Yes
3. amplify push finds no changes or api

**Expected behavior**
GraphQL API is created as expected and push works fine.

**Output**
Failing output. Note: `spawn code ENOENT`

```
$ amplify add api 
? Please select from one of the below mentioned services GraphQL
? Provide API name: RestaurantAPI
? Choose an authorization type for the API API key
? Do you have an annotated GraphQL schema? No
? Do you want a guided schema creation? Yes
? What best describes your project: Single object with fields (e.g., “Todo” with ID, name, desc
ription)
? Do you want to edit the schema now? Yes
Please edit the file in your editor: /Users/midwo1/Documents/wsc/amplify-app/amplify/backend/api/RestaurantAPI/schema.graphql
? Press enter to continue spawn code ENOENT

$ amplify push
Current Environment: dev
| Category | Resource name      | Operation | Provider plugin   |
| -------- | ------------------ | --------- | ----------------- |
| Auth     | amplifyapp70e10f3b | No Change | awscloudformation |
 
No changes detected
```

**Desktop:**
 - OS: Mac and Windows
 - Amplify CLI: tried on 2.0.0 and 1.12.0

**Additional context**
Visual Code not correctly set on terminal for one machine (Mac) but was working on Windows (Eg: `$ code .`)
```
$ code 
-bash: code: command not found
$ vscode
-bash: vscode: command not found
```

**Fix**
Skip guided schema creation.
```
$ amplify add api 
? Please select from one of the below mentioned services GraphQL
? Provide API name: RestaurantAPI
? Choose an authorization type for the API API key
? Do you have an annotated GraphQL schema? No
? Do you want a guided schema creation? No
? Provide a custom type name MyType
Creating a base schema for you...
 
The following types do not have '@auth' enabled. Consider using @auth with @model
       - MyType
Learn more about @auth here: https://aws-amplify.github.io/docs/cli-toolchain/graphql#auth 
 
 
GraphQL schema compiled successfully.
 
Edit your schema at /Users/midwo1/Documents/wsc/amplify-app/amplify/backend/api/RestaurantAPI/schema.graphql or place .graphql files in a directory at /Users/midwo1/Documents/wsc/amplify-app/amplify/backend/api/RestaurantAPI/schema
Successfully added resource RestaurantAPI locally
 
Some next steps:
"amplify push" will build all your local backend resources and provision it in the cloud
"amplify publish" will build all your local backend and frontend resources (if you have hosting category added) and provision it in the cloud
 
$ amplify status 
Current Environment: dev
 
| Category | Resource name      | Operation | Provider plugin   |
| -------- | ------------------ | --------- | ----------------- |
| Api      | RestaurantAPI      | Create    | awscloudformation |
| Auth     | amplifyapp70e10f3b | No Change | awscloudformation |
```

**Comment by yuth** [comment]
This should be fixed by #2163. We released this fix this morning. Please feel free to comment or re-open the issue if the problem still persists

**Comment by ywroh** [comment]
@gsans I also checked the spawn code ENOENT error that occurred in the amplify-cli open-editor execa as follows.
![스크린샷 2020-12-16 오전 12 49 58](https://user-images.githubusercontent.com/39849622/102238318-f4a36700-3f38-11eb-8c97-666cef8a59b6.png)


'Do you want to edit the schema now?' I solved the problem by saying'No' to the answer to this question.
![스크린샷 2020-12-16 오전 12 54 03](https://user-images.githubusercontent.com/39849622/102238551-37fdd580-3f39-11eb-8d86-3e6a1a035aab.png)
